### PR TITLE
Create separate finding for Microsoft RDP port

### PR DESCRIPTION
**Pull request by stephanie0x00**
### Changes
This PR updates the findings DB and tweaks the bit for port classification such that when a Microsoft RDP port is detected, a separate finding is created for this. From a security perspective it is good to know if this point is present in the attack surface. 

If an open RDP port is detected, the finding is given a 'medium' severity, as there is no immediate risk per se. A recommendation severity would also be possible, however RDP is often the root cause for ransomware attacks, thus that feels a bit low. Other ideas for an appropriate severity are welcome. 

### Issue link
Partial solution for #3837

### Demo

image

### QA notes
- Perform a nmap scan against a server with an open RDP port, e.g. run `nc -l 3389` locally or use the IP from the screenshot (this is a Google server).  
- Perform nmap tcp scan against the IP. 
- Somehow the finding Type didn't resolve properly, thus the finding remained pending. This can be resolved by triggering the Finding type boefje manually. 

---

### Code Checklist

- [x] All the commits in this PR are properly PGP-signed and verified.
- [x] This PR only contains functionality relevant to the issue.
- [ ] I have written unit tests for the changes or fixes I made.
- [x] I have checked the documentation and made changes where necessary.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

- [ ] Tickets have been created for newly discovered issues.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Comment by noamblitz** [comment]
I think its a good idea to add this to the Question as well since all other port configs are also asked there. For the rest this looks good!

**Comment by stephanie0x00** [comment]
I've updated the Question schema :+1:

**Review by ammar92 — COMMENTED** [review]
Nice work! Just a few tiny things to complete this PR:
- Since one of the recommendation strings has been updated; it affected the `test_port_classification_tcp_22` test in Octopoes. Meaning "should not be open" should be changed to "should possibly not be open."
- Format error in the JSON file, can be fixed by running pre-commit

**Comment by Rieven** [comment]
## Checklist for QA:

- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
  - [x] I have managed to pass the onboarding flow
  - [x] Objects and Findings are created properly
  - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.
- [x] I checked the logs for errors and/or warnings and made issues where necessary

### What works:

It works

File: octopoes/bits/port_classification_ip/port_classification_ip.py [hunk 1]
```diff
@@ -79,6 +83,18 @@ def run(input_ooi: IPPort, additional_oois: list, config: dict[str, Any]) -> Ite
                     ooi=ip_port.reference,
                     description=f"Port {port}/{protocol.value} is a database port and should not be open.",
                 )
+        elif (protocol == Protocol.TCP or protocol == Protocol.UDP) and port in microsoft_rdp_ports:
```
Comment (underdarknl) [inline octopoes/bits/port_classification_ip/port_classification_ip.py:86]:
```suggestion
        elif  port in microsoft_rdp_ports:
```

**Pull request merged**